### ISSUE TITLE
Add keyboard shortcut to hide all windows in the "Windows" pane

### ISSUE THREAD

**Issue by uncertainquark** [elementary/switchboard-plug-keyboard#162]
Currently, it's only possible to minimize the current window using the keyboard shortcut "Super+H"
![screenshot from 2018-05-03 16-05-23](https://user-images.githubusercontent.com/22585732/39572280-37aa8582-4eec-11e8-8748-fe313a62605a.png)

However, there is no option to hide/minimize all the windows at once. The unused shortcut "Super+D" can be assigned to that, as it also falls in line with the traditionally expected paradigm of showing the Desktop. 

At the least, it could be an option but I reckon, it will be quite useful as a default too.

**Comment by Astro-City** [comment]
Hi.  

I don't agree with this.  

I think that the Super+D should be the default shortcut to go to the last workspace (+). This way seems more in line with elementary work flow where the minimize isn't a windows button.  

From what i read, even the plank minimize will go away in the future.  

If someone want to have a minimize all windows shorcut, just follow this stems:  
- Install the wmctrl package 

- create an executable file  named wmctrl on the home folder with the following text:  

```
#!/bin/sh
if	wmctrl -m | grep "mode: ON"; then
	exec wmctrl -k off
else
	exec wmctrl -k on
fi
```

- Create a custom shortcut with the file location.

**Comment by Blast-City** [comment]
I noticed that there doesn't seem to be a "bug free" way to minimize all windows (wmctrl, plank docklet) so I'm gonna retract my opinion since there is also people who use Static workspaces and this is necessary.

Maybe a hidden gsettings that does the same as gnome, but also triggers the wingpanel to stop the tiled/maximized state (black opaque color). #453

**Comment by uncertainquark** [comment]
Navigating to a new empty workspace on `Win`+`D` also works. Aim is to get all windows out of the way quickly and have a shortcut that is expected by users coming from other systems.

**Comment by Blast-City** [comment]
> Navigating to a new empty workspace on `Win`+`D` also works. Aim is to get all windows out of the way quickly and have a shortcut that is expected by users coming from other systems.

I agree ([#279](https://github.com/elementary/gala/issues/279)), but the problem is users who use static workspaces.

**Comment by uncertainquark** [comment]
But elementary OS doesn't have static workspaces by default, nor does it have configurable workspaces, right? In that case, we need not worry about such cases as they are not catered to by design, much like how double click in Files app isn't catered to intentionally.
